.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: Push Validations
+
+on:
+  push:
+    branches: [master, dev, feature/*, bugfix/*, chore/*]
+
+jobs:
+  lint-test-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [12.x]
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Use Node.js ${{ matrix.node_version }}
+        uses: actions/setup-node@v1
+        with:
+          node_version${{ matrix.node_version}}
+
+      - name: install
+        run: npm ci
+
+`     - name: run lint
+        run: npm lint
+
+      - name: run test
+        run: npm test
```

Comment:
We use angular.
Are you sure this will be okay?
I think we should use ng test instead.
Could you please check out this one and others?

Reply:
It's specified in the documentation, that we should use npm command https://angular.io/guide/testing#configure-project-for-github-actions